spec/unit/provider/package/dpkg_spec.rb

```diff
@@ -52,8 +51,8 @@
   end
 
   before(:each) do
-    allow(provider).to receive(:shell_out!).with("dpkg-deb -W #{new_resource.source}", timeout: 900).and_return(dpkg_deb_status)
-    allow(provider).to receive(:shell_out).with("dpkg -s #{package}", timeout: 900).and_return(dpkg_s_status)
+    allow(provider).to receive(:shell_out!).with("dpkg-deb -W #{source}", timeout: 900).and_return(dpkg_deb_status)
```

Comment:
Maybe I was thinking of some other trick, but now I can't find where I used it :disappointed: